packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSameSizeAfterReopening.st

```diff
@@ -0,0 +1,8 @@
+testing
+testAlgernonPositioningWindowIsSameSizeAfterReopening
+	| point oldPosition|
+	point := 0 @ 0.
```

Comment:
You can remove the "point" here.

Reply:
We had problems adding the point without having it saved in a variable.

Reply:
Right, but adding a point with the value 0@0 won't do that much. 😃 You can remove the point variable and summation altogether.